matplotlib-stubs/collections.pyi

```diff
@@ -0,0 +1,11 @@
+from .artist import Artist
+from .cm import ScalarMappable
+
+class Collection(Artist, ScalarMappable):
+    pass
+
+class _CollectionWithSizes(Collection):
```

Comment:
I don't think we need to include private classes in our stubs

Reply:
I don't need it as a parent class for PolyCollection?

Reply:
We can just say

```python
class PolyCollection(Collection):
     pass
```
right? skipping the middle man.

Reply:
While I doubt few, if any, people will make use of them, what about the methods defined in `_CollectionWithSizes`?

Reply:
You can leave it as it is. But in general I don't think we should spend our time replicating the minutia of the class hierarchies in matplotlib.